index.js

```diff
@@ -84,21 +72,34 @@
 //.
 //. ### Usage
 //.
+//. #### With native types
+//.
 //. ```javascript
-//. var Identity = require('my-package').Identity;
-//. var type = require('sanctuary-type-identifiers');
-//.
-//. type(null);         // => 'Null'
-//. type(true);         // => 'Boolean'
-//. type([1, 2, 3]);    // => 'Array'
-//. type(Identity);     // => 'Function'
-//. type(Identity(0));  // => 'my-package/Identity'
+//. > var type = require('.');
```

Comment:
Ahhh, it has access to the variables defined in the scope where the code comment exists? That's clever!